project-base/src/Shopsys/ShopBundle/Resources/views/Front/Inline/Product/productFlags.html.twig

```diff
@@ -0,0 +1,3 @@
+{% import '@ShopsysShop/Front/Inline/Product/productFlagsMacro.html.twig' as productFlags %}
```

Comment:
please explain this new macro in commit message or extract it into a standalone commit

Reply:
👍 I will do the change on rebase to not mess with history right now

Reply:
I described the FlagsExtension itself in the commit message as there is no new macro